### PR TITLE
Fixed typo in readme concerning IComparable JSon serializer

### PR DESCRIPTION
**Pull request by dschuermans**
Also added the Structural Analysis Format project to list of UnitsNet users

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -434,3 +434,15 @@ https://www.harringtonhoists.com<br>
...
+
+#### Structural Analysis Format - SDK project
+<img src="https://gblobscdn.gitbook.com/spaces%2F-M__87HTlQktMqcjAf65%2Favatar-1620901174483.png?alt=media" height="35" />
+
+> The Structural Analysis Format (SAF) has been created to allow structural engineering applications to exchange data using a straight forward and simple to understand format.  
+> While inspired by IFC, SAF has its benefits that it's **easily modifyable** by the end-user _(it's an xlsx file)_, **well documented** and **easy to understand**.  
+> UnitsNet is used by the SDK provided by SCIA to facilitate import / export between metric & imperial systems
+
+https://www.saf.guide  
+https://github.com/StructuralAnalysisFormat _(SDK will soon be available under this org. when it's ready, soon&trade;)_
```
Comment (angularsen) [inline README.md:446]: I would leave out parentheses and just keep the link here.

**Review by angularsen — CHANGES_REQUESTED** [review]
Great! Just a minor comment.

File: README.md (hunk 1 above)
Comment (dschuermans) [reply README.md:446]: Ah okay, shall I also update the link to the actual repo (which is currently private)? I expect it to be made public in a few weeks

**Comment by angularsen** [comment]
Sure, I think that is fine 👍

File: README.md (hunk 1 above)
Comment (dschuermans) [reply README.md]: Done!

**Pull request merged**